bindings.rs

```diff
@@ -28,32 +29,63 @@ fn main() {
         false
     };
 
-    // if !dynamic {
-    // }
+    if fetch_xmlsec_version()
+        >= (Version {
+            major: 1,
+            minor: 3,
+            patch: 0,
+            pre: Default::default(),
+            build: Default::default(),
+        })
+    {
+        println!("cargo:rustc-cfg=xmlsec_1_3");
+    }
+
     println!("cargo:rustc-link-lib=xmlsec1-openssl"); // -lxmlsec1-openssl
     println!("cargo:rustc-link-lib=xmlsec1"); // -lxmlsec1
     println!("cargo:rustc-link-lib=xml2"); // -lxml2
     println!("cargo:rustc-link-lib=ssl"); // -lssl
     println!("cargo:rustc-link-lib=crypto"); // -lcrypto
 
-    if !path_bindings.exists() {
```

Comment:
Dropped this check since we should be depending on cargo's rerun logic for the build script rather than a path check like this. As written, it leads to stale bindings that need to be manually removed or `cargo clean`ed away.